### PR TITLE
DAOS-9138 cart: Quiet -DER_GRPVER ERR messages

### PR DESCRIPTION
**Pull request by liw**
As requested, quiet common -DER_GRPVER ERRs to DBUGs.

Signed-off-by: Li Wei <wei.g.li@intel.com>

### CODE REVIEW

File: src/include/cart/api.h [hunk 1]
```diff
@@ -2249,6 +2249,19 @@ void crt_swim_fini(void);
...
+ *
+ * \param[in] err                an error
+ *
+ * \return                       true or false
+ */
+static inline bool
+crt_quiet_error(int err)
+{
+	return err == -DER_GRPVER;
+}
```
Comment (johannlombardi) [inline src/include/cart/api.h:2263]: Does it really need to be visible through the external API?

Comment (frostedcmos) [reply src/include/cart/api.h:2263]: it wont be as it is static

**Review by frostedcmos — APPROVED** [review]

File: src/include/cart/api.h (hunk 1 above)
Comment (liw) [reply src/include/cart/api.h:2263]: Because I'd like to call this function from daos_quiet_error, to avoid duplicating the crt list in daos.

Comment (johannlombardi) [reply src/include/cart/api.h:2263]: @frostedcmos i think that you are confused by the meaning of static inline.

@liw got it

**Review by johannlombardi — APPROVED** [review]

File: src/include/cart/api.h (hunk 1 above)
Comment (frostedcmos) [reply src/include/cart/api.h:2263]: @johannlombardi ah, i thought this was in .c file

**Pull request merged**